auto-cpufreq.py

```diff
@@ -221,8 +221,8 @@ def set_performance():
         #print("\n" + "-" * 60 + "\n")
         footer(79)
     else:
-        print("Load optimal, setting turbo boost: off")
-        s.run("echo 1 > /sys/devices/system/cpu/intel_pstate/no_turbo", shell=True)
+        print("Load optimal, setting turbo boost: on")
+        s.run("echo 0 > /sys/devices/system/cpu/intel_pstate/no_turbo", shell=True)
         #print("\n" + "-" * 60 + "\n")
```

Comment:
If you put "turbo on" in else statement, then turbo will never turn off in performance, which is contradicting with [what I said in my comment to this MR](https://github.com/AdnanHodzic/auto-cpufreq/pull/34#issuecomment-573408002).

Reply:
But I changed the governor "performance" to "powersave". powersave governor will not push the clock to turbo state if it isn't really a necessity but I really don't sure which one gives the good performance. My laptop has 1.8 GHz without TB, 3.9 with TB.

Reply:
I made another comment regarding setting switching to `performance` while in `powersave` (discharging battery).

Please correct me if I'm wrong, but you're saying while in "powersave", turbo never goes on for you? Because for me it certainly does.

Reply:
I'm not saying that. When on the charging mode we will run `set_performance()` function. If there isn't any or little load of CPU, we should provide to the user `powersave` governor because we don't need the high clock speed, but if there is a need for turboboost, `powersave` will definitely handle that, and helps to the PC for staying cool. Beside of that, if there is a noticeable load, we give to the user the power and activate `performance` governor.

Reply:
As I discussed in [my other comment](https://github.com/AdnanHodzic/auto-cpufreq/pull/34#discussion_r367795655), at this stage we really shouldn't change "governors" based on cpu usage/system load.

I would rather refine that turbo is turned on/off based on these in a more refined way. Let's take care of the other comment I made and merge this PR.